midpoint/install/system-requirements/index.adoc

```diff
@@ -190,135 +196,133 @@ This is important to keep in mind when choosing the right HA strategy.
 
 In the center of the schema, the largest rectangle represents a virtual machine, usually running Linux, with the xref:/midpoint/install/bare-installation/distribution/#purpose-and-quality[basic required set of tools] installed.
+MidPoint runs on this virtual machine.
+The database repository preferably runs on a separate virtual machine, albeit possibly on the same physical server.
 
 ==== Shared Database Repository
 
-For the database repository (DB), an existing DB server is usually used.
-You have access to the DB using the SQL via the TCP/IP from the midPoint server. 
-Do not forget to configure the firewalls to enable communication over the used ports—5432 typically for PostgreSQL, 1433 for MS SQL, 1521 for Oracle, etc.
+For the database repository (DB), you can use an existing DB server or set up a new dedicated one.
+The midPoint server needs to have access to it using SQL via TCP/IP.
+Do not forget to configure the firewalls to enable communication over the used ports.
 
 The database needs to be configured prior to midPoint installation so that the person who installs midPoint can configure the midPoint instance correctly.
-In the single-node mode, the DB can be placed on a separate DB server.
+If you deploy in containers, see xref:/midpoint/install/containers/#deployment-schema[] for details.
 
 ==== Notification System
 
 If e-mail notifications are needed, access to the SMTP server and an account with send privileges is required.
 If you need SMS notifications, you also need access to an SMS gateway and have the account privileges to send SMS.
 
-==== Secure Remote Deployment
+==== Secure Access to Your Deployment
+
+If your setup requires access to your midPoint deployment over the public Internet, you need to secure the communication between end users' computers and the midPoint instance.
+One of the options is to use a virtual private network (VPN) for everyone who needs to access midPoint.
+VPN provides a tunnel from the user's machine to midPoint (or the load balancer before it).
 
-If the deployment is remote (i.e., you don't host it on-site), you need to secure the communication between end-user workstations and the midPoint instance.
-One of the options is to use a virtual private network (VPN) for each team member who needs to access midPoint.
-VPN provides a tunnel from the user's workstation to midPoint (or the load balancer before it).
+==== Troubleshooting Access to Nodes
 
-A separate SSH access for troubleshooting purposes is advised, so that you have a chance to examine the situation in case the VPN or the load balancer fail.
+You can set up an SSH access to your midPoint nodes.
+You can use it to access midPoint configuration files and logs in case something goes wrong.
+If you deploy on xref:/midpoint/install/containers/kubernetes/[Kubernetes] and use xref:/midpoint/operations-manual/#_syslog_logging[remote logging], you do not need SSH access to individual nodes to manage, diagnose, and fix them remotely.
```

Comment:
This depends on the deployment. With proper Kubernetes architecture it might be true but it's not enforced.